tests/flow/test_function_calls.py

```diff
@@ -987,4 +1003,27 @@ def test35_min_max(self):
         query = "UNWIND [1, 2, '1' ,'2' ,[1] ,[2] ,1 ,2, '1', '2', NULL, True] AS x RETURN max(x), min(x)"
         actual_result = graph.query(query)
         expected_result = [[2, [1]]]
-        self.env.assertEquals(actual_result.result_set, expected_result)
\ No newline at end of file
+        self.env.assertEquals(actual_result.result_set, expected_result)
+
+    def test36_division_inputs(self):
+        # Validate integer dividend division by 0
+        query = "RETURN 3 / 0"
+        try:
+            actual_result = graph.query(query)
+        except redis.ResponseError as e:
+            self.env.assertContains("Division by zero", str(e))
+        
+        # Validate floating-point dividend division by 0
+        query = "RETURN 1.0 / 0"
+        actual_result = graph.query(query)
+        self.env.assertTrue(math.isinf(actual_result.result_set[0][0]))
```

Comment:
Why is there a difference between integer and floating point?
the former yields an error while the latter returns infinity.

Should we check for positive and negative infinity ?
e.g. `-1 / 0.0` and `1 / 0.0` ?

Reply:
Ok. those tests were added in a3523cc. 
Tests with both operands equal to zero were added too.

Reply:
I think the idea is like so:
When performing `x/0` the division is undefined as `x` is being divided by an **absolute** 0.
While `x/0.0` is either `+inf` or `-inf` depending on `x`
As `0.0` isn't necessarily an absolute `0` but a **really small number** who's almost reaching `0`